pootle/apps/pootle_language/views.py

```diff
@@ -108,3 +112,26 @@ def language_admin(request, language):
     }
     return admin_permissions(request, language.directory,
                              'languages/admin/permissions.html', ctx)
+
+
+@get_path_obj
+@permission_required('administrate')
+def language_characters_admin(request, language):
+    if request.method == 'POST':
+        form = LanguageSpecialCharsForm(request.POST, instance=language)
```

Comment:
Less boilerplate and one level less of nesting by using the [following pattern](http://www.pydanny.com/the-easy-form-views-pattern-controversy.html):
```python
from = SpecialCharsForm(request.POST if request.method === 'POST' or None,
                        instance=language)
```
Using CBVs makes this irrelevant as well. Use your best judgement.